### PR TITLE
Added StitchIfwi support for WHL/CFL platforms

### PR DESCRIPTION
**Pull request by raghavag**
This patch will add support for StitchIfwi.py to generate
Boot Guard enabled IFWI images for WHL/CFL platforms

Signed-off-by: Raghava Gudla <raghava.gudla@intel.com>

### CODE REVIEW

File: Platform/CoffeelakeBoardPkg/Script/StitchLoader.py [hunk 1]
```diff
@@ -235,17 +229,36 @@ def LocateComponent(Root, Path):
...
+def AddPlatformData (InputFile, RgnLen, PlatformData = None):
     Fd = open(InputFile, "rb")
     InputData = bytearray(Fd.read())
     Fd.close()
 
     InputLen = len(InputData)
-    if RgnLen < InputLen:
-        Fatal("Input binary size(0x%08x) cannot exceed %s region size(0x%08x) !" % (InputLen, Rgn[0], RgnLen))
+    if RgnLen is not 0:
+        if RgnLen < InputLen:
```
Comment (mauricema) [inline Platform/CoffeelakeBoardPkg/Script/StitchLoader.py]: where is InputLen defined ?

File: Platform/CoffeelakeBoardPkg/Script/StitchLoader.py [hunk 2]
```diff
@@ -235,17 +229,36 @@ def LocateComponent(Root, Path):
...
+          PlatDataOffset = Stage1A.Offset + Stage1A.Length - 12
+          c_uint32.from_buffer (InputData, PlatDataOffset).value = PlatformData
+        print "Platform data was patched for %s" % Path
+      return InputData
+
+def ReplaceRegion (IfwiData, Rgn, InputFile, TailPos, PlatformData = None):
+    RgnPos = Rgn[1]
+    RgnLen = Rgn[2]
+
+    InputData = AddPlatformData(InputFile, RgnLen, PlatformData)
```
Comment (mauricema) [inline Platform/CoffeelakeBoardPkg/Script/StitchLoader.py]: you need to add InputLen = len(InputData)

Comment (raghavag) [reply Platform/CoffeelakeBoardPkg/Script/StitchLoader.py]: Done

File: Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+    sys.stdout.flush()
+    try:
+        CmdArgs = shlex.split(Cmd)
+        if os.name == 'nt':
+            Shell = True
+            if CmdArgs[0].startswith('./'):
+                CmdArgs[0] = CmdArgs[0][2:]
+        else:
+            Shell = False
+        #Proc = subprocess.Popen(CmdArgs, shell = Shell, cwd=Cwd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (mauricema) [inline Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py]: should we just remove the commented line ?

**Pull request merged**